toolkit/tools/internal/timestampcsvparser/timestampcsvparser.go

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Parses the timestamps stored in CSV files and print them
+// to the terminal at the end of the build.
+
+package timestampcsvparser
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+var (
+	timeArray [][]string // A 2-D array used to store all timestamps from CSVs.
+	// A list of CSV files to be parsed.
+	files []string
+)
+
+// Reads a CSV file and appends line by line to array.
+func CSVToArray(filename string) {
```

Comment:
Why is this a public function? I think we're only calling it in this file.

Reply:
Will address via switch to .json format.